frontend/amundsen_application/api/metadata/v0.py

```diff
@@ -776,6 +776,8 @@ def _get_dashboard_metadata(*, uri: str, index: int, source: str) -> None:
         url = f'{app.config["METADATASERVICE_BASE"]}{DASHBOARD_ENDPOINT}/{uri}'
 
         response = request_metadata(url=url, method=request.method)
+        if response.status_code != 200:
+            raise RuntimeError("Metadata service errored " + response.json()["message"])
```

Comment:
We don't raise errors generally we catch the errors and then put them in the API response like we do for tables https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/api/metadata/v0.py#L172

Reply:
have to make sure the error get's logged also

Reply:
If you want to run with these changes go for it. But I made this PR  2 years ago and I can't really shift to working back on this project right now.